repos/exporter/lib/database.ts

```diff
@@ -173,3 +182,75 @@ export const getFareDayEnd = async (noc: string): Promise<string | undefined> =>
 
     return queryResults[0]?.time;
 };
+
+export const getPointToPointProducts = async (): Promise<
+    {
+        id: number;
+        lineId: string;
+        matchingJsonLink: string;
+        startDate: string;
+        endDate: string;
+    }[]
+> => {
+    const queryInput = `      
+            SELECT id, lineId, matchingJsonLink, startDate, endDate
+            FROM products
+            WHERE lineId <> ''
+        `;
+    return await executeQuery(queryInput, []);
+};
+
+export const getDirectionAndStopsByLineIdAndNoc = async (lineId: string, noc: string): Promise<DirectionAndStops[]> => {
+    const nocCodeParameter = replaceInternalNocCode(noc);
+
+    const serviceQuery = `
+        SELECT DISTINCT pl.fromAtcoCode, pl.toAtcoCode, ps.direction, os.id as serviceId
+        FROM txcOperatorLine AS os
+        JOIN txcJourneyPattern AS ps ON ps.operatorServiceId = os.id
+        JOIN txcJourneyPatternLink AS pl ON pl.journeyPatternId = ps.id
+        WHERE os.nocCode = ? AND os.lineId = ? AND os.dataSource = 'bods'
+    `;
+
+    try {
+        return await executeQuery(serviceQuery, [nocCodeParameter, lineId]);
+    } catch (error) {
+        throw new Error(`Could not get journey patterns from Aurora DB.`);
+    }
+};
+
+declare interface ResultSetHeader {
```

Comment:
not sure ive ever seen declare - is it different to export here?

Reply:
export is preferred outside of declaration files